pkg/service/restore/worker_schema.go

```diff
@@ -1,12 +1,13 @@
-// Copyright (C) 2022 ScyllaDB
+// Copyright (C) 2023 ScyllaDB
 
-package backup
+package restore
 
 import (
 	"context"
 	"path"
```

Comment:
I suggest to change the name of the file.
Please replace `worker_schema` with `schema_worker`.

`worker_schema` defines that schema related logic of `worker` struct is kept here.